e2e/editable-grid.e2e-spec.ts

```diff
@@ -270,3 +270,93 @@ describe('Editable grid', () => {
 
   });
 });
+
+describe('Editable grid, complex cells', () => {
+
+  // selectors
+  const selectCell = '.ag-body-viewport [aria-colindex="2"]';
+  const selectCellTrigger = '.ag-body-viewport [aria-colindex="2"] .ag-picker-field-display';
+  const selectList = '.ag-select-list';
+  const editButton = '#edit-btn';
+
+  let currentTheme: string;
+  let currentThemeMode: string;
+
+  async function selectTheme(theme: string, mode: string): Promise<void> {
```

Comment:
Minor - but could we just move the first describe block’s `selectTheme` and `getScreenshotName` above the first `describe` block? Then we wouldn’t need this duplicate code.